operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakController.java

```diff
@@ -45,9 +45,10 @@
 import java.util.Optional;
 import java.util.Set;
 
+import static io.javaoperatorsdk.operator.api.reconciler.Constants.NO_FINALIZER;
 import static io.javaoperatorsdk.operator.api.reconciler.Constants.WATCH_CURRENT_NAMESPACE;
 
-@ControllerConfiguration(namespaces = WATCH_CURRENT_NAMESPACE)
+@ControllerConfiguration(namespaces = WATCH_CURRENT_NAMESPACE, finalizerName = NO_FINALIZER)
```

Comment:
Deletion of the Deployment was always getting stuck with finalizers, we are not using them ATM so I think is correct to disable them at all. cc. @vmuzikar